pkg/vadmin/restore_points_query_vc.go

```diff
@@ -17,9 +17,62 @@ package vadmin
 
 import (
 	"context"
+	"fmt"
+
+	vops "github.com/vertica/vcluster/vclusterops"
+	"github.com/vertica/vcluster/vclusterops/vstruct"
+	"github.com/vertica/vertica-kubernetes/pkg/net"
+	"github.com/vertica/vertica-kubernetes/pkg/vadmin/opts/restorepoints"
+	ctrl "sigs.k8s.io/controller-runtime"
 )
 
-func (v *VClusterOps) ListRestorePoints(_ context.Context) error {
-	v.Log.Info("start restore points query")
-	return nil
+// RestartNode will restart a subset of nodes. Use this when vertica has not
+// lost cluster quorum. The IP given for each vnode may not match the current IP
+// in the vertica catalogs.
+func (v *VClusterOps) ListRestorePoints(ctx context.Context, opts ...restorepoints.Option) (ctrl.Result, error) {
```

Comment:
We use ListRestorePoints when we are going to call ShowRestorePoints. I think we need some consistency here. Wherever we have `list`, we should use `show` instead.